src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -519,21 +522,30 @@ private static void ValidateDeprecated(JsonSchema schema, string name, JToken to
 
         private void ValidateDocsetUnique(JsonSchema schema, JObject map)
         {
+            var monikers = _monikerProvider?.GetFileLevelMonikers(ErrorBuilder.Null, t_filePath.Value!).ToList();
+            if (monikers == null || !monikers.Any())
+            {
+                monikers = new List<string>(new[] { string.Empty });
```

Comment:
why we need a string empty? will an empty string array work?

Reply:
We use empty string here as default moniker. Added comment on this.